src/plone/restapi/services/actions/get.py

```diff
@@ -34,4 +47,20 @@ def reply(self):
                     'icon': action['icon'],
                 })
             data[category] = category_action_data
-        return data
+        return {'actions': data}
```

Comment:
@csenger @sneridagh I had to wrap the data into actions to make it work with the expansion mechanism. I would have expected a test failure in test_actions but that did not happen at all. Need to investigate this further...

Reply:
@csenger @sneridagh forget about what I wrote. This is specific to the expansion mechanism. I got confused. I think this is an indicator that we might want to re-think the entire expansion API design at some point...Anyways, I think except the docs we are good now...